packaging/systemd/configure-ovs.sh

```diff
@@ -282,6 +282,10 @@ EOF
       if [ -n "$ipv6_addr_gen_mode" ]; then
         extra_if_brex_args+="ipv6.addr-gen-mode ${ipv6_addr_gen_mode} "
       fi
+      ipv4_ignore_auto_dns=$(nmcli --get-values ipv4.ignore-auto-dns conn show ${old_conn})
+      if [ -n "$ipv4_ignore_auto_dns" ]; then
+        extra_if_brex_args+="ipv4.ignore-auto-dns ${ipv4_ignore_auto_dns} "
+      fi
```

Comment:
let's add also ipv6 support

https://bugzilla.redhat.com/show_bug.cgi?id=1261428

ipv6.ignore-auto-dns and let's avoid future bugs :), dns settings are exposed via router advertisement, or DHCPv6